### DISCUSSION TITLE
Add width to a DataColumn/DataRow

### DISCUSSION THREAD

**Discussion by ArthurSHausmann** [flet-dev/flet#4429] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

Could you please add a Width parameter for DataColumns and Datarows? Im having a lot of trouble making an resonsive table that fits in my screen, please ty ty ty

### Screenshots

![image](https://github.com/user-attachments/assets/c664b192-a37f-4769-8841-6b4555a885a8)

**Comment by ndonkoHenri** [comment]
Wrap your `DataTable` in a `Row`, to make it horizontally scrollable.

Also, to make your cells more adaptive, set `DataTable.data_row_max_height=float("inf")` - ([discussion](https://github.com/flet-dev/flet/issues/3444), [docs](https://flet.dev/docs/controls/datatable/#data_row_max_height))
